src/DocBlock/Annotation.php

```diff
@@ -226,6 +226,15 @@ public function setTypes(array $types)
         $this->clearCache();
     }
 
+    public function getNormalizedTypes()
+    {
+        $normalized = array_map('strtolower', $this->getTypes());
```

Comment:
maybe we want this refactored to not rely on the function name, but I'll leave that to a second reviewer :)

Reply:
yes, we don't want to call functions by string with their name, as it's impossible to analyse with static code analysis
please use anonymous function as a wrapper here

Reply:
Would be good if there were a fixer for that ;)

Reply:
I did it this way because I had seen that in `NativeConstantInvocationFixer`. Want me to fix it there as well, or just here?

Reply:
Fixed here, let me know if you want me to fix the other one too.